### DISCUSSION TITLE
Suspense Support

### DISCUSSION THREAD

**Discussion by rondered** [reduxjs/redux-toolkit#2724] [Ideas]
Do you have any estimation to when we will get a full suspense support to queries?

**Comment by phryneas** [comment]
Since suspense for data fetching on the client is not yet a feature that is officially supported by React itself, we are waiting for their go.

There is ongoing experimentation in https://github.com/reduxjs/redux-toolkit/pull/2245 but until the React team tells us how suspense for data fetching is actually going to be used, we are just guessing.
